schema.prisma

```diff
@@ -1,678 +1,693 @@
+  sourceCampaign   Campaign       @relation(fields: [sourceCampaignId], references: [id])
+  sourceVault      Vault          @relation(fields: [sourceVaultId], references: [id])
+  @@map("withdrawals")
+}
+
+model BankAccount {
+  id                String            @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  status            BankAccountStatus @default(new)
+  /// IBAN number
+  ibanNumber        String            @db.VarChar(34)
+  /// Name of account holder
+  accountHolderName String            @map("account_holder_name")
+  /// Company or individual
+  accountHolderType AccountHolderType @map("account_holder_type")
+  /// Bank name
+  bankName          String?           @map("bank_name") @db.VarChar(50)
+  /// Bank Identification Code, BIC/SWIFT code
+  bankIdCode        String?           @map("bank_id_code") @db.VarChar(50)
+  fingerprint       String?           @db.VarChar(100)
+  createdAt         DateTime          @default(now()) @map("created_at") @db.Timestamptz(6)
+  updatedAt         DateTime?         @updatedAt @map("updated_at") @db.Timestamptz(6)
+  withdraws         Withdrawal[]
+  @@map("bank_accounts")
+}
+
+/// Pay for something from a given vault
+model Expense {
+  id           String        @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  type         ExpenseType
+  status       ExpenseStatus
+  currency     Currency      @default(BGN)
+  amount       Int           @default(0)
+  vaultId      String        @map("vault_id") @db.Uuid
+  deleted      Boolean       @default(false)
+  description  String?
+  documentId   String?       @map("document_id") @db.Uuid
+  approvedById String?       @db.Uuid
+  vault        Vault         @relation(fields: [vaultId], references: [id])
+  approvedBy   Person?       @relation(fields: [approvedById], references: [id])
+  document     Document?     @relation(fields: [documentId], references: [id])
+  @@map("expenses")
+}
+
+model Document {
+  id          String       @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  type        DocumentType
+  name        String       @db.VarChar(100)
+  filename    String       @db.VarChar(100)
+  filetype    String?      @db.VarChar(3)
+  description String?      @db.VarChar(200)
+  /// Data storage source url
+  sourceUrl   String       @map("source_url")
+  /// Person who uploaded the document
+  ownerId     String       @map("owner_id") @db.Uuid
+  owner       Person       @relation(fields: [ownerId], references: [id])
+  expenses    Expense[]
+  @@map("documents")
+}
+model BankTransactionsFile {
+  id         String           @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  filename   String           @db.VarChar(200)
+  mimetype   String           @db.VarChar(50)
+  bankTransactionsFileId String           @map("bank_transactions_file_id")  @db.VarChar(50)
+  personId   String           @map("person_id") @db.Uuid
+  type       BankTransactionsFileType

```

Comment:
You may add a `@default(xml)` or `@default(other)` if you think this will be appropriate.


Also the whole `schema.prisma` file has been updated with different encoding than before. See diff in PR